src/main/java/org/gridsuite/explore/server/ContingencyListService.java

```diff
@@ -93,9 +93,9 @@ public void setActionsServerBaseUri(String actionsServerBaseUri) {
             .log(ROOT_CATEGORY_REACTOR, Level.FINE);
     }
 
-    public Mono<Void> newScriptFromFiltersContingencyList(UUID id, String scriptName, UUID newId) {
-        String path = UriComponentsBuilder.fromPath(DELIMITER + ACTIONS_API_VERSION + "/filters-contingency-lists/{id}/new-script/{scriptName}?newId={newId}")
-            .buildAndExpand(id, scriptName, newId)
+    public Mono<Void> newScriptFromFiltersContingencyList(UUID id, UUID newId) {
+        String path = UriComponentsBuilder.fromPath(DELIMITER + ACTIONS_API_VERSION + "/filters-contingency-lists/{id}/new-script?newId={newId}")
```

Comment:
Maybe the opposite should be done, change insertNewScriptFromFilter to specify script uuid as a request param instead of a path variable to allow script creation without specified id.

Reply:
Modifications done in FilterService and also in the filter-server